src/theme/components/Terminal/Ironfish.js

```diff
@@ -7,18 +7,18 @@ export default [
 CLI for running and interacting with an Iron Fish node
 
 VERSION
-  ironfish/0.1.57 darwin-arm64 node-v18.12.1
+  ironfish/0.1.62 darwin-arm64 node-v18.12.1
 
 USAGE
   $ ironfish [COMMAND]
 
 TOPICS
   blocks      Show the block header of a requested hash or sequence
   chain       Manage the blockchain
-  config      Show and edit the node configuration
+  config      Print out the entire config
   migrations  List all the migration statuses
-  miners      Manage an Iron Fish miner
-  peers       Manage the peers connected to this node
+  miners      Start a mining pool that connects to a node
```

Comment:
This is correct but this description is a bug, it's just taking it from the first command in the topic

Reply:
yup, reverting this